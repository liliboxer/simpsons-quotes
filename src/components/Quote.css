.Quote {
  margin: 10px;
  width: 30vw;
  height: 50vh;
  padding: 10px;
  text-align: center;
  box-shadow: 0px 0px 5px 5px rgb(22, 177, 92);
}