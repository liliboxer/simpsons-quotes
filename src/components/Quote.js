import React from 'react';
import PropTypes from 'prop-types';

function Quote({ characterName, quote, characterImage }) {
  return (
    <>
      <h4>{characterName}</h4>
      <p>{quote}</p>
      <img src={characterImage}></img>
    </>
  );

}

Quote.propTypes = {
  characterName: PropTypes.string.isRequired,
  quote: PropTypes.string.isRequired,
  characterImage: PropTypes.string.isRequired
};

export default Quote;

