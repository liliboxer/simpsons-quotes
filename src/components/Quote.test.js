import React from 'react';
import { shallow } from 'enzyme';
import Quote from './Quote';

describe('Quote component', () => {
  it('renders Quote', () => {
    const wrapper = shallow(<Quote characterName={''} quote={''} characterImage={''}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
