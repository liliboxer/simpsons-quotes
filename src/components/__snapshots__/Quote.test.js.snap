// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote component renders Quote 1`] = `
<section
  className="Quote"
>
  <h4 />
  <p />
  <img
    src=""
  />
</section>
`;
